examples/blaze/src/main/scala/com/example/http4s/blaze/demo/server/Module.scala

```diff
@@ -14,10 +14,9 @@ import org.http4s.client.Client
 import org.http4s.server.HttpMiddleware
 import org.http4s.server.middleware.{AutoSlash, ChunkAggregator, GZip, Timeout}
 
-import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration._
 
-class Module[F[_]](client: Client[F])(implicit F: Effect[F], S: Scheduler) {
+class Module[F[_]](client: Client[F])(implicit F: ConcurrentEffect[F], S: Scheduler, T: Timer[F]) {
```

Comment:
What is the scheduler here? Is it the fs2 one? In that case isn't it superseded by the timer?

Reply:
`Scheduler` is deprecated in fs2-1.0.  If we can replace it in http4s-0.19, that would be ideal.  Anywhere we have both is suspect.

I'm fine doing that in a subsequent PR.